environment.yml

```diff
@@ -25,6 +26,7 @@ dependencies:
   - sphinx_rtd_theme
   - sphinx-argparse
   - nbsphinx
+  - numba::numba
```

Comment:
Why install numba from the numba namespace? It's on conda forge

Reply:
I did this because the pip install project.toml uses a different source for the installation. In my test pip used 0.52 while conda-forge used 0.51. For consistency I though it would be better to get the latest version for both (or enforce a specific version for both).
This line can be removed otherwise.

Reply:
Please use the standard version from conda-forge, we should avoid having too many channels in the environment file.

Reply:
It should also be updated very quicklyu after releases.

Reply:
@gabemery this comment is not yet addressed.